packages/browser-capabilities/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 <!-- ## Unreleased -->
 <!-- Add new, unreleased changes here. -->
+* Upated Firefox 67 support for modules (specifically because of dynamic import)
```

Comment:
Upated -> Updated

Reply:
I UP4T3 ALL DA TIM VAN DER LIPPE

Reply:
I literally have a typo in the commit message where I fix the typo 🙈